asreview/webapp/src/ProjectComponents/AnalyticsComponents/NumberCard.js

```diff
@@ -84,35 +86,37 @@ export default function NumberCard(props) {
             </CardContent>
           </Card>
         </Grid>
-        <Grid item xs={12} sm={12}>
-          <Card className="number-card" elevation={2}>
-            <CardContent>
-              <Stack spacing={2} className="number-card-content">
-                <Typography
-                  className="number-card-content-text"
-                  variant={!props.mobileScreen ? "subtitle1" : "subtitle2"}
-                  sx={{ color: "text.secondary" }}
-                >
-                  Irrelevant records since last relevant
-                </Typography>
-                <Typography
-                  className="number-card-content-numeral"
-                  variant={!props.mobileScreen ? "h4" : "h5"}
-                >
-                  <NumberFormat
-                    value={
-                      showNumber()
-                        ? props.progressQuery.data["n_since_last_inclusion"]
-                        : 0
-                    }
-                    displayType="text"
-                    thousandSeparator
-                  />
-                </Typography>
-              </Stack>
-            </CardContent>
-          </Card>
-        </Grid>
+        {hasPriorKnowledge() && (
+          <Grid item xs={12} sm={12}>
+            <Card className="number-card" elevation={2}>
+              <CardContent>
+                <Stack spacing={2} className="number-card-content">
+                  <Typography
+                    className="number-card-content-text"
+                    variant={!props.mobileScreen ? "subtitle1" : "subtitle2"}
+                    sx={{ color: "text.secondary" }}
+                  >
+                    Irrelevant records since last relevant
+                  </Typography>
+                  <Typography
+                    className="number-card-content-numeral"
+                    variant={!props.mobileScreen ? "h4" : "h5"}
+                  >
+                    <NumberFormat
+                      value={
+                        showNumber()
+                          ? props.progressQuery.data["n_since_last_inclusion"]
+                          : 0
+                      }
+                      displayType="text"
+                      thousandSeparator
+                    />
+                  </Typography>
+                </Stack>
+              </CardContent>
+            </Card>
+          </Grid>
+        )}
```

Comment:
Although I proposed the solution in the issue like this, it might be better to display a different message or `-` instead of the number rather than a conditional display. What do you think?